tests/integration/memcached/test_basic.php

```diff
@@ -46,6 +46,7 @@
     [{"name":"Datastore/allOther"},                                   [15, "??", "??", "??", "??", "??"]],
     [{"name":"Datastore/Memcached/all"},                              [15, "??", "??", "??", "??", "??"]],
     [{"name":"Datastore/Memcached/allOther"},                         [15, "??", "??", "??", "??", "??"]],
+    [{"name":"Datastore/instance/Memcached/memcached/11211"},          [1, "??", "??", "??", "??", "??"]],
```

Comment:
Where do the values of `memcached` as host and `11211` come from?

Reply:
memcached is set up in the docker-compose [here](https://github.com/newrelic/newrelic-php-agent/blob/main/docker-compose.yaml#L80) and [here](https://github.com/newrelic/newrelic-php-agent/blob/main/docker-compose.yaml#L33). The default port is 11211, but we set the environment variable [here](https://github.com/newrelic/newrelic-php-agent/blob/main/tests/include/config.php#L28).

Reply:
If this information is available in the environment, then please update these tests to use environment variables instead of hardcoding these values. Here's an example of what I mean:
```suggestion
    [{"name":"Datastore/instance/Memcached/ENV[MEMCACHE_HOST]/ENV[MEMCACHE_PORT]"},          [1, "??", "??", "??", "??", "??"]],
```

Reply:
MEMCACHE_PORT is a PHP variable, not an environment variable, and as far as I can tell there is therefore no easy way to replace the hard coded value? Because it is hard coded in many places, is the default, and is unlikely to ever change, I am inclined to leave it as is

Reply:
I don't like the fact that port is hardcoded but changing this is not part of this PR. This is good enough.